django_ninja_crudl/renderers.py

```
@@ -0,0 +1,32 @@
+from typing import Any, override
+
+from django.core.serializers.json import DjangoJSONEncoder
+from ninja.renderers import JSONRenderer
+from pydantic import BaseModel
+from pydantic.networks import AnyUrl
+from pydantic_core import Url
+
+__all__ = ["CrudlJSONEncoder", "CrudlJSONRenderer"]
+
+
+class CrudlJSONEncoder(DjangoJSONEncoder):
```

Ruff (0.11.9)

Missing docstring in public class

(D101)